storage/src/cache/worker.rs

```diff
@@ -264,9 +264,8 @@ impl AsyncWorkerMgr {
                 AsyncPrefetchMessage::FsPrefetch(state, blob_cache, req) => {
                     let _ = mgr2.prefetch_sema.acquire().await;
                     if state.load(Ordering::Acquire) > 0 {
-                        rt.spawn(async move {
-                            let _ = Self::handle_fs_prefetch_request(mgr2.clone(), blob_cache, req)
-                                .await;
+                        rt.spawn_blocking(move || {
+                            let _ = Self::handle_fs_prefetch_request(mgr2.clone(), blob_cache, req);
```

Comment:
Any explanation why we need to change to a blocking spawn?

Reply:
Yes. The created tokio runtime has two types of queues which is either processing blocking tasks or non-blocking tasks. For blocking tasks, we should use `spawn_blocking` to drop it to blocking task queue.